templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,16 @@
-# Press Enter to send
+# Enable or disable Press Enter to send
 
 By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+on your keyboard.
 
-![Box checked](/static/images/help/enter-box-checked.png)
+If you want to disable this feature, simply uncheck the
+**Press Enter to send** option below the compose box.
 
-You will now see a **Send** button below the compose field looking
-like this.
+You will now see a **Send** button below the compose field looking like
```

Comment:
We have `looking like this` here, but there is no screenshot to show it. What's the reasoning for removing it?